gpu-jobs.md

```diff
@@ -188,21 +182,29 @@ Note that HTCondor will make sure your job has access to the GPU -- you
 shouldn't need to set any environmental variables or other options
 related to the GPU, except what is needed inside your code.
 
+It is possible to request multiple GPUs. Before doing so, make sure you're 
+using code that can utilize multiple GPUs and then submit a test to confirm 
+this works before submitting a bigger job. Also keep track of how long jobs 
+are running versus waiting; the time you save by using multiple GPUs may be 
+not worth the extra time that the job will likely wait in the queue. 
+
 ## B. Use the GPU Lab Servers (recommended)
 
-To accept the CHTC GPU Lab policies and opt-in to use these GPUs, add the
+To use CHTC's shared use GPUs, you need to opt-in to the GPU Lab. To 
```

Comment:
Maybe figure out how to have this show up sooner?

Reply:
Same comments as previously -- would require a restructuring of the guide.